src/test/bip32_tests.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2013-2020 The Bitcoin Core developers
+// Copyright (c) 2013-2021 The Bitcoin Core developers
```

Comment:
Nit: we usually update the copyright years once a year in bulk

Reply:
Yes, but will this hurt? Script will just ignore this one, right?

Reply:
No, that's why it's just a nit :)